modules/ROOT/pages/dw-core-functions-mapobject.adoc

```diff
@@ -97,3 +97,24 @@ output application/xml
 
 Helper function that allows `mapObject` to work with null values.
 
+=== Example
+
+Using the previous example, you can test that if the input of the `mapObject` is `null`, the output result is `null` as well. In XML `null` values by default are written as empty tags, that can be changed using the writer property `writeNilOnNull=true`.
```

Comment:
In XML, `null`values are written as empty tags. You can change these values by using...